File: Logitude/Logitude.BL/QuoteModel/DataContracts/QuoteTemplateSettingData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Logitude.BL.QuoteModel.DataContracts
{
    public class QuoteTemplateSettingData
    {
        [DataMember]
        public List<PricesFieldSettings> PricesPackagesTableSettings { get; set; }

        [DataMember]
        public List<PricesFieldSettings> PricesContainersTableSettings { get; set; }

        [DataMember]
        public string PricingPackagesSplitChargeType { get; set; }

        [DataMember]
        public string PricingContinersSplitChargeType { get; set; }

        [DataMember]
        public bool ShowUnitsContainers { get; set; }
    }

    public class PricesFieldSettings
    {
        [DataMember]
        public string Name { get; set; }
        [DataMember]
        public string Code { get; set; }
        [DataMember]
        public int Index { get; set; }
        [DataMember]
        public bool InUse { get; set; }
    }
}

File: Logitude/AngularModules/AngularModules/Quote/DataContracts/QuoteTemplateSettingData.ts

export class QuoteTemplateSettingData {
    public PricesPackagesTableSettings: PricesFieldSettings[];
    public PricesContainersTableSettings: PricesFieldSettings[];
    public PricingPackagesSplitChargeType: string;
    public PricingContinersSplitChargeType: string;
}

export class PricesFieldSettings {
    public Name: string;
    public Code: string;
    public Index: number;
    public InUse: boolean;
    public Key: string;
    public Show: string;
    public IsShowArrowUpDown: boolean;
}
